src/prototypes/draft-email-textarea/demo/demo.twig

```diff
@@ -0,0 +1,55 @@
+{# The `o-container` wrappers are here only for demo purposes #}
+<div class="o-container o-container--pad o-container--prose">
+	<div class="o-container__content o-rhythm o-rhythm--default o-rhythm--generous-headings">
+		{# Draft email message #}
+		<textarea
+			id="email-draft"
+			class="c-input u-pad-1 js-elastic-textarea"
+			spellcheck="false"
+		>
+			{{ draftMessage }}
+		</textarea>
```

Comment:
Can we omit the whitespace so the textarea has enough room all around?

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/2109/commits/7b529eef8ea148f1c0f488a5211a91cdc04d9904